.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    steps:
+      - checkout
+      - run:
+          name: Start containers
+          command: |
+            set -x
+            docker-compose up -d
+      - run:
+          name: Set up the project
+          command: |
+            docker-compose exec web composer campaign:build || true
+            docker-compose exec web chmod ugo+rw web/sites/default -R
+      - run:
+          name: Run the test
+          command: |
+            docker-compose exec web sh -c 'cd web && drush pm-uninstall cookieconsent toolbar -y'
+            docker-compose exec web vendor/bin/behat -n
+            docker-compose exec web vendor/bin/behat -ns rest
```

Comment:
@Saphyel why are you not using https://github.com/comicrelief/campaign/blob/master/ci/travis/run-tests.sh ? like you do for the dberrors script? Maybe we should refactor that and just put these in the `ci/` directory so both Travis / Circle can use these?

Reply:
@pvhee because I was testing the junit thing, and travis is not able to do anything with the reports.